### ISSUE TITLE
Doesn't work with subgrid

### ISSUE THREAD

**Issue by lts20050703** [isaacHagoel/svelte-dnd-action#594]
### Describe the bug
I have a grid whose rows are drag and drop to reorder using svelte-dnd-action. Because there can only be one element in the #each block, I create a wrapper div with `grid-column: 1 / -1;`, `display: grid`, and `grid-template-rows: subgrid` so that the wrapper div reuse the same track size as the container. However, when I drag and drop it looks like svelte-dnd-action ghost element is not inheriting the parent grid tracks, which makes the element looks out of place.

### Reproduction

https://www.sveltelab.dev/k47f4u9wghd3yip

The code is based on svelte-dnd-action's [basic example](https://github.com/isaacHagoel/svelte-dnd-action?tab=readme-ov-file#basic-example) with the following changes:
- the `items` variable is declared with `$state` to make it reactive like how it would in Svelte 4.
- `on:*` directives changed to `on*` properties
- Styles/Layout: Mentioned above.

### System Info

```shell
  System:
    OS: Linux 6.5 KDE neon 6.0 6.0
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
    Memory: 5.88 GB / 15.40 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 22.5.1 - ~/.local/share/nvm/v22.5.1/bin/node
    Yarn: 1.22.22 - ~/.local/share/nvm/v22.5.1/bin/yarn
    npm: 10.8.2 - ~/.local/share/nvm/v22.5.1/bin/npm
    pnpm: 9.4.0 - ~/.local/share/nvm/v22.5.1/bin/pnpm
    bun: 1.1.16 - ~/.bun/bin/bun
  Browsers:
    Brave Browser: 127.1.68.134
  npmPackages:
    svelte: ^5.0.0-next.206 => 5.0.0-next.206
```

### Severity

annoyance

**Comment by isaacHagoel** [comment]
Modifying this function should do the trick.
https://github.com/isaacHagoel/svelte-dnd-action/blob/master/src/helpers/styler.js#L94
I am open to PRs if you have time to look into it

**Comment by isaacHagoel** [comment]
yeah it doesn't deep copy the styles, we tried that and it caused problems.
when i think about it you might be able to either style the dragged element
directly (using a global css rule on it's id) or using the
'transformDraggedElement' option

**Comment by lts20050703** [comment]
Thank you! `transformDraggedElement` is a life-saver! I have found a solution that I'm happy with!

I was planning to make a PR to make svelte-dnd-action add the dragged element to the parent dndzone instead of root but it still doesn't work. It turns out `position: fixed` by definition made that element not belong to the grid, so it would never work in the first place (https://stackoverflow.com/a/58144759/16118261)

Right now svelte-dnd-action doesn't copy `grid-*` CSS properties so I used the `transformDraggedElement` option to add the `grid-template-columns` property to the dragged element.

For future readers, I can't make every element in the #each block a grid because I have inputs that auto-grow/resize to fit the number of characters. I need to use the value `auto` so that the grid looks like a grid, not a bunch of misaligned columns.

ELI5: The grid needs to look like this
```
| A | 12345 | B |
| C |   0   | D |
```
and not
```
| A | 12345 | B |
|  C  | 0 |  D  |
```
and I can't hard code columns' widths, because the middle column's inputs auto-grow/resize to fit the number of characters
